src/ifcwrap-csharp/utils/typemaps_out.i

```diff
@@ -0,0 +1,202 @@
+// %typemap(out) aggregate_of_instance::ptr {
```

Comment:
This code is commented out, either:
- Remove it
- Un-comment it
- Comment why it is fine to be commented-out